packages/celotool/src/cmds/geth/start.ts

```diff
@@ -0,0 +1,180 @@
+import yargs from 'yargs'
+
+import { addCeloGethMiddleware } from 'src/lib/utils'
+import { AccountType, getPrivateKeysFor, getValidatorsInformation } from '../../lib/generate_utils'
+import { runGethNodes } from '../../lib/geth'
+import { GethRunConfig } from '../../lib/interfaces/geth-run-config'
+import { GethArgv } from '../geth'
+
+export const command = 'start'
+export const describe = 'command for running geth'
+
+interface StartArgv extends GethArgv {
+  networkId: string
+  syncMode: string
+  mining: boolean
+  blockTime: number
+  nodekeyhex: string
+  minerGasPrice: number
+  port: number
+  rpcport: number
+  wsport: number
+  verbosity: number
+  verbose: boolean
+  amount: number
+  purge: boolean
+  withProxy: boolean
+  ethstats: string
+}
+
+export const builder = (argv: yargs.Argv) => {
+  return addCeloGethMiddleware(argv)
+    .option('network-id', {
+      type: 'string',
+      description: 'network id',
+      default: '1101',
+    })
+    .option('sync-mode', {
+      choices: ['full', 'fast', 'light', 'ultralight'],
+      demandOption: true,
+    })
+    .option('mining', {
+      type: 'boolean',
+      description: 'Is mining enabled',
+      default: false,
+    })
+    .option('miner-gas-price', {
+      type: 'number',
+      description: 'Mining gas price',
+      default: 0,
+    })
+    .option('port', {
+      type: 'number',
+      description: 'Port',
+      default: 30303,
+    })
+    .option('rpcport', {
+      type: 'number',
+      description: 'HTTP-RPC server listening port',
+      default: 8545,
+    })
+    .option('wsport', {
+      type: 'number',
+      description: 'WS-RPC server listening port',
+      default: 8546,
+    })
+    .option('amount', {
+      type: 'number',
+      description: 'Amount of nodes to run',
+      default: 1,
+    })
+    .option('with-proxy', {
+      type: 'boolean',
+      description: 'Start with proxy in front',
+      default: false,
+    })
+    .option('verbosity', {
+      type: 'number',
+      description: 'Verbosity level',
+      default: 5,
+    })
+    .option('verbose', {
+      type: 'boolean',
+      description: 'Verbose',
```

Comment:
To distinguish it from verbosity above
```suggestion
      description: 'Command verbosity flag',
```

Reply:
done